src/core/localization/translations/en/common.json

```diff
@@ -120,8 +120,14 @@
       "reset_limits_to_default": "Reset limits to default",
       "exclude": "Exclude",
       "as_values_on_limits": "As values on limits",
-      "natural_logarithm": "Logarithm",
-      "square_root": "Square root",
+      "transformations": {
+        "no_transformation": "No transformation",
+        "natural_logarithm": "Logarithm",
```

Comment:
So is this a natural log or log10? Natural has e as base.

Reply:
That is a natural log, it's for an old transformation option (I have a question about whether we need it for tomorrow discussion)